crates/forge-runner/src/lib.rs

```diff
@@ -68,9 +68,10 @@ pub fn maybe_save_trace_and_profile(
     }) = result
     {
         if execution_data_to_save.is_vm_trace_needed() {
-            let trace_path = save_trace_data(name, trace_data)?;
+            let name = sanitize_filename::sanitize(name.replace("::", "_"));
```

Comment:
This is a breaking change, please mention it in the changelog

Reply:
It's only changing the names of the files, the directory remains the same. I can mention it, but I don't think we ever documented the format of the names anywhere.

Reply:
Yes, but potentially it can be used as an input to profiler or coverage in some workflows